dronecan/sensors/rc/1141.RCLinkStats.uavcan

```diff
@@ -0,0 +1,39 @@
+#
+# This definition describes metadata and statistics for a RF control link.
+# It provides information regarding a link's signal quality, as well as transmitter
+# power, and antenna information. It accomodates single-radio, and dual-radio
+# link architectures. It provides information for both uplink (control unit to airborne
+# receiver), and downlink (airborne receiver to control unit) data.
+#
+
+uint8 STATUS_RSSI_VALID = 1 # RSSI field is valid
+
+# Uplink - received signal strength, in decibel-milliwatts (RSSI dBm) for antennas 1 and 2 respectively. 
+# Ranges from # -255 to 0. (An integer value of 100 means RSSI is -100dBm)
+uint8 uplink_rssi_1
+uint8 uplink_rssi_2
+
+# Link quality, on a scale from 1 to 255. Is a proxy for packet loss. 0 means data is unavailable. 1 and 255 are mapped to 0% and 100% of packets received.
+uint8 uplink_link_quality
+
+# Signal-to-noise ratio.
+int8 uplink_snr
+
+# This is an implementation-specific enum. May represent the OTA protocol used,
+# and transmission rate.
+uint8 rf_mode
+
+# This is an implementation-specific enum describing the current transmitter nominal
+# power level.
+uint8 uplink_tx_power
+
+# Statistics for downlink data, ie telemetry. See their respective `uplink` fields for
+# descriptions.
+uint8 downlink_rssi
```

Comment:
uplink has rrsi1 and rssi2 - how come only one here? I ask because the mavlink proposal is mirrored.

Reply:
Done